math/octave-devel/Portfile

```diff
@@ -2,253 +2,38 @@
 
 PortSystem          1.0
 PortGroup           compilers  1.0
-PortGroup           muniversal 1.0
-PortGroup           compiler_blacklist_versions 1.0
 PortGroup           linear_algebra 1.0
-PortGroup           select     1.0
+PortGroup           cxx11 1.1
+PortGroup           qt4 1.0
 
 name                octave-devel
 categories          math science
 platforms           darwin
 license             GPL-3+
 maintainers         mcalhoun openmaintainer
 description         a high-level language for numerical computations
-long_description    \
-    GNU Octave is a high-level language, primarily intended for numerical \
-    computations. It provides a convenient command-line interface for \
-    solving linear and nonlinear problems numerically.
+long_description    ${description}
 
 homepage            http://www.gnu.org/software/octave
 
-#fetch.type          hg
-#hg.url              http://hg.savannah.gnu.org/hgweb/octave/
 
-set version_release 4.0.3
-set version_rc      4.2.0-rc4
-set version_tip     4.2.0-rc4
+### Fetch tip of dev tree - change version to today's date
 
-if { ${name} eq ${subport} } {
-    version         ${version_tip}
-    revision        3
-    hg.tag          e078f5607762
+fetch.type          hg
+hg.url              http://hg.savannah.gnu.org/hgweb/octave/
+hg.tag              -1
+version             20171102
```

Comment:
The version should be 4.x.y_20171102, so that there's no need to increase the epoch afterwards.

Reply:
Yes. I just looks Ken's version string. We really should be using 4.3.0+_ 20171102, as octave  tip identifies itself as version 4.3.0+